static/css/responsive.css

```
@@ -193,6 +193,25 @@
     outline-offset: 2px;
 }
 
+/* ── High contrast mode ─── */
+@media (prefers-contrast: high) {
+    * {
+        border-color: currentColor !important;
```

Stylelint (17.12.0)

[error] Expected "currentColor" to be "currentcolor" (value-keyword-case)

(value-keyword-case)